src/neural-network.js

```diff
@@ -119,12 +119,42 @@ export default class NeuralNetwork {
     }
   }
 
+  /**
+   *
+   * @returns boolean
+   */
+  get isRunnable(){
+    if(!this.runInput){
+      console.error('Activation function has not been initialized, did you run train()?');
+      return false;
+    }
+
+    const checkFns = [
+      'sizes',
+      'outputLayer',
+      'biases',
+      'weights',
+      'outputs',
+      'deltas',
+      'changes',
+      'errors',
+    ].map(c => this[c]).every(fn => fn !== null);
```

Comment:
If you change this to a filter, and check for length, you'd have the properties that failed

Reply:
added, that is much cleaner/better